lib/cli/behaviour/tasks/add-gitignore.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const path = require('path');
+const append = require('../../../utils/append');
+
+const ignore = [];
+
+module.exports = function gitignore(settings) {
+  return append(path.resolve(settings.dir, '.gitignore'), ignore.join('\n') + '\n');
+};
```

Comment:
This is appending some nothing to the `.gitignore` and can probably be completely removed.

If it's going to continue to exist it should probably find some commonality with the `init` version.